src/theme/components/icon.ts

```diff
@@ -40,6 +40,10 @@ const size = {
     height: "var(--nypl-space-m)",
     width: "var(--nypl-space-m)",
   },
+  default: {
```

Comment:
The `default` size should go to 100% width.  That is how `Image` and `Logo` treat `default`.

Reply:
Sorry, I forgot I had this in here.  This is obviously a requested change.  Does the thumbs-up mean you will make that change in this PR?

Reply:
Yes, that I'll make a change. For this one, this is width 100%:
<img width="1028" alt="Screen Shot 2022-03-10 at 12 55 51 PM" src="https://user-images.githubusercontent.com/1280564/157737762-13a35437-2e30-4aaa-b31d-85f24cf43fbc.png">

and this is width and height 100%:
<img width="1042" alt="Screen Shot 2022-03-10 at 12 55 42 PM" src="https://user-images.githubusercontent.com/1280564/157737817-01520ac3-79ef-48d7-9bd5-780a451bf2fd.png">

It didn't seem right to me so that's why I made it similar to `medium`. Which one should it be?